### PR TITLE
Peter Ridolfi 2022 Fellow

### PR DESCRIPTION
**Pull request by peterridolfi**
[peterridolfi.md](https://github.com/iris-hep/iris-hep.github.io-source/files/8682281/peterridolfi.md)
Peter-Ridolfi

### CODE REVIEW

**Comment by henryiii** [comment]
Could you reduce the size of the image? 2 MB is pretty hefty.

File: pages/fellows/peterridolfi.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+presentations:
+  - title: "<Presentation Title"
+    date: "Presentation Date"
+    url: <Presentation materials link>
+    meeting: <Meeting name>
+    meetingurl: <Meeting url - indico link, etc.>
+    recordingurl: <Recording url> (Optional)
+    focus-area: <Focus Area - ia,ssl,ssc,doma,as,osglhc>
+current_status: >
+  A placeholder for status updates
```
Comment (henryiii) [inline pages/fellows/peterridolfi.md]:
```suggestion
```

Please do not include placeholders.

File: pages/fellows/peterridolfi.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  start: 2022-5-16
+  end: 2022-8-12
+photo: /assets/images/team/Peter-Ridolfi.jpg
+institution: Univeristy of Pittsburgh
+e-mail: pbr11@pitt.edu
+project_title: Designing and implementing a converting tool for statistical models between pyhf and CMS combine
+project_goal:
+    The main goal of this project is to develop a tool that actively identifies and accomodates similarities and nuances between pyhf and CMS Combine models. The tool must be able to convert the given parameters form one type of model to the other type, while maintaining a high degree of accuracy between the actual functionality and predictions that are made by the models. Additionally, the tool should be able to give feedback concerning expected differences in these outputs based on the parameters that are given, and the limitations of the tool itself. Ultimately, the tool will serve as another method for better understanding the models that are created and the predictions that they make.
+mentors:
+  - Matthew Feickert - (Univeristy of Illinois at Urbana-Champaign)
```
Comment (henryiii) [inline pages/fellows/peterridolfi.md]: Please look up and use shortnames if possible.

File: pages/fellows/peterridolfi.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  - Alex Held - (University of Wisconsin-Madison)
+  - Kyle Cramer - (New York University)
```
Comment (henryiii) [inline pages/fellows/peterridolfi.md]:
```suggestion
  - matthewfeickert
  - alexander-held
  - cranmer
```

**Comment by henryiii** [comment]
In the future, please make a branch before making a PR. It's much easier to manipulate a branch with a unique name vs. having to push to master/main on a remote. Thanks!

File: pages/fellows/peterridolfi.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+shortname: peterridolfi
+permalink: /fellows/peterridolfi.html
+fellow-name: Peter Ridolfi
+title: Peter Ridolfi - IRIS-HEP Fellow
+active: True
+dates:
+  start: 2022-5-16
+  end: 2022-8-12
+photo: /assets/images/team/Peter-Ridolfi.jpg
+institution: Univeristy of Pittsburgh
```
Comment (henryiii) [inline pages/fellows/peterridolfi.md]:
```suggestion
institution: University of Pittsburgh
```

File: pages/fellows/peterridolfi.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+e-mail: pbr11@pitt.edu
+project_title: Designing and implementing a converting tool for statistical models between pyhf and CMS combine
+project_goal:
+    The main goal of this project is to develop a tool that actively identifies and accomodates similarities and nuances between pyhf and CMS Combine models. The tool must be able to convert the given parameters form one type of model to the other type, while maintaining a high degree of accuracy between the actual functionality and predictions that are made by the models. Additionally, the tool should be able to give feedback concerning expected differences in these outputs based on the parameters that are given, and the limitations of the tool itself. Ultimately, the tool will serve as another method for better understanding the models that are created and the predictions that they make.
```
Comment (henryiii) [inline pages/fellows/peterridolfi.md]:
```suggestion
    The main goal of this project is to develop a tool that actively identifies and accommodates similarities and nuances between pyhf and CMS Combine models. The tool must be able to convert the given parameters form one type of model to the other type, while maintaining a high degree of accuracy between the actual functionality and predictions that are made by the models. Additionally, the tool should be able to give feedback concerning expected differences in these outputs based on the parameters that are given, and the limitations of the tool itself. Ultimately, the tool will serve as another method for better understanding the models that are created and the predictions that they make.
```

**Comment by peterridolfi** [comment]
Yeah sure, my bad I didn’t check the size.

**Pull request merged**